openedx/core/djangoapps/user_api/views.py

```diff
@@ -65,24 +65,32 @@ def get(self, request):
 
         """
         form_desc = FormDescription("post", reverse("user_api_login_session"))
+        email_or_username = settings.FEATURES.get('ENABLE_LOGIN_BY_EMAIL_OR_USERNAME', False)
 
         # Translators: This label appears above a field on the login form
         # meant to hold the user's email address.
         email_label = _(u"Email")
+        if email_or_username:
+            email_label = _(u"Email or username")
```

Comment:
I think it's better to make the label `Email or Username`. It looks better to properly capitalize on the word `Username`